models/devicecommand.go

```diff
@@ -5,10 +5,33 @@
 
 package models
 
+import "maps"
+
 type DeviceCommand struct {
 	Name               string
 	IsHidden           bool
 	ReadWrite          string
 	ResourceOperations []ResourceOperation
 	Tags               map[string]any
 }
+
+func (dc DeviceCommand) Clone() DeviceCommand {
+	cloned := DeviceCommand{
+		Name:               dc.Name,
+		IsHidden:           dc.IsHidden,
+		ReadWrite:          dc.ReadWrite,
+		ResourceOperations: dc.ResourceOperations,
+		Tags:               dc.Tags,
```

Comment:
Since these two fields will be cloned below, do we need these two lines?

Reply:
No, it's redundant, removed.